ftw/simplelayout/contenttypes/profiles/default/types/ftw.simplelayout.ContentPage.xml

```diff
@@ -33,6 +33,7 @@
     <property name="behaviors">
         <element value="ftw.simplelayout.interfaces.ISimplelayout" />
         <element value="ftw.simplelayout.opengraph.interfaces.IOpenGraphSupport" />
+        <element value="ftw.simplelayout.interfaces.IContentPageShowTitle" />
```

Comment:
@raphael-s We don't need this behavior by default. When needed, we can activate it in the customer project.

So I propose to:

- [ ] Do not enable it in `ftw.simplelayout.ContentPage.xml`
- [ ] Make sure the changes in `simplelayout.pt` do not break when the behavior is not available
- [ ] Document the existence of the behavior in the Readme.